README.md

```diff
@@ -1,4 +1,99 @@
 Sailjail permissions
 ====================
-
 Permissions for [Sailjail](https://github.com/sailfishos/sailjail/).
+These are [firejail](https://firejail.wordpress.com/) profile files with
+Sailjail specific metadata format.
```

Comment:
Could be also Sailfish specific as Sailjail itself doesn't really use it for anything. It's used only on UI.

Reply:
+1 for using "sailfish" rather than "sailjail"